base/ltpara.dtx

```diff
@@ -819,6 +819,18 @@
   \@@_handle_indent:
 % \the \everypar           % <--- done differently below
 }
+%    \end{macrocode}
+% \changes{v1.0n}{2024/06/21}
+%         {Append \cs{everypar} toks to \cs{g_@@_standard_everypar_tl},
+%          rollback 2023/06/01 (gh/1386)}
+%    \begin{macrocode}
+%<latexrelease>\cs_set:Npn \@@_tmp:w #1#2#3#4#5 { }
+%<latexrelease>\tl_gput_right:Nx \g_@@_standard_everypar_tl {
```

Comment:
`\tl_gpur_right:Ne`

Reply:
Do you also suggest to change other two `\tl_gput_right:Nx`s in `ltpara.dtx`?

Reply:
@josephwright  I think it's Ok to use `e` (as @muzimuzhi  has already changed it) but this is rollback so rolling back to what it was would have been more natural  I think?

Reply:
Ah this is rollback.

I just checked that `\tl_gput_right:Ne` was not provided until l3kernel 2023-10-10 (see latex3/latex3@fe0eaaa7fbfc59ce9226c7613036a76ef9694c94). So maybe I should keep using `x` in that rollback code for 2023/06/01.

Reply:
For rollback I suggest we stick to roll back to what we had back then and not do some thing like x to e (or whatever) even if it appears to be safe. Any unnecessary change would at least alter a test with `\CheckCommand` (even if that is unlikely). Of course they may be a few cases where we want to divert from that general rule, but as a best practice I think that is better.

Reply:
@muzimuzhi  sorry can you revert back to `x`

Reply:
Done. I dropped the 'x to e' commit https://github.com/latex3/latex2e/commit/1c23fde6c347c8d9f0e59657642e2256962819cd.